tests/expression.rs

```diff
@@ -0,0 +1,24 @@
+#![feature(plugin,const_fn)]
+#![plugin(stainless)]
+
+describe! addition {
+    before_each {
+        let x = 5;
+        let y = 6;
+        for _ in 0..5 {
+        }
+    }
+
+    it "should add 5 and 6 together" {
```

Comment:
Could you rename this to something that says what this test case is about? Maybe `"gets executed before the after_each block even if it ends with an expression"` or something similar (I'm not a native speaker).